dolphin-dnn/README.md

```diff
@@ -90,6 +101,10 @@ The following functions are supported.
 * Absolute: `abs`
 * Softmax: `softmax`
 
+##### Loss functions
+The following loss functions are supported.
+* CrossEntropy: `crossEntropy`
```

Comment:
Letter case in configuration file does not matter since our code is case-insensitive for strings in the configuration file. `crossEntropy` is better to read than `crossentropy`. I'll change configuration samples to use `crossEntropy`.